gapis/api/vulkan/extensions/khr_surface.api

```diff
@@ -55,7 +55,21 @@
 
 @extension("VK_KHR_surface")
 enum VkColorSpaceKHR: u32 {
-  VK_COLORSPACE_SRGB_NONLINEAR_KHR = 0x00000000,
+  VK_COLORSPACE_SRGB_NONLINEAR_KHR            = 0x00000000,
+  VK_COLOR_SPACE_DISPLAY_P3_NONLINEAR_EXT     = 0x3b9c6041,
+  VK_COLOR_SPACE_EXTENDED_SRGB_LINEAR_EXT     = 0x3b9c6042,
+  VK_COLOR_SPACE_DISPLAY_P3_LINEAR_EXT        = 0x3b9c6043,
+  VK_COLOR_SPACE_DCI_P3_NONLINEAR_EXT         = 0x3b9c6044,
+  VK_COLOR_SPACE_BT709_LINEAR_EXT             = 0x3b9c6045,
+  VK_COLOR_SPACE_BT709_NONLINEAR_EXT          = 0x3b9c6046,
+  VK_COLOR_SPACE_BT2020_LINEAR_EXT            = 0x3b9c6047,
+  VK_COLOR_SPACE_HDR10_ST2084_EXT             = 0x3b9c6048,
+  VK_COLOR_SPACE_DOLBYVISION_EXT              = 0x3b9c6049,
+  VK_COLOR_SPACE_HDR10_HLG_EXT                = 0x3b9c604a,
+  VK_COLOR_SPACE_ADOBERGB_LINEAR_EXT          = 0x3b9c604b,
+  VK_COLOR_SPACE_ADOBERGB_NONLINEAR_EXT       = 0x3b9c604c,
+  VK_COLOR_SPACE_PASS_THROUGH_EXT             = 0x3b9c604d,
+  VK_COLOR_SPACE_EXTENDED_SRGB_NONLINEAR_EXT  = 0x3b9c604e,
```

Comment:
From https://www.khronos.org/registry/vulkan/specs/1.1-extensions/html/vkspec.html#VkColorSpaceKHR, it seems following are not included
 ```  
    VK_COLOR_SPACE_DISPLAY_NATIVE_AMD = 1000213000,
    VK_COLORSPACE_SRGB_NONLINEAR_KHR = VK_COLOR_SPACE_SRGB_NONLINEAR_KHR,
    VK_COLOR_SPACE_DCI_P3_LINEAR_EXT = VK_COLOR_SPACE_DISPLAY_P3_LINEAR_EXT, 
```